app/controllers/finders_controller.rb

```diff
@@ -124,9 +124,15 @@ def results
   end
 
   def all_facets
-    return @facets if defined?(@facets)
-
-    FacetsBuilder.new(content_item:, search_results:, value_hash: filter_params).facets
+    @all_facets ||= FacetsBuilder.new(
```

Comment:
Quick double-check - I assume it's safe to memoize this (ie we're expecting this method to always return the same values no matter how many times it's called during any one call to the action that references it?)

Reply:
I think so: I think [this change](https://github.com/alphagov/finder-frontend/commit/51ef79134a1bcab9bfa7de2b14aaaa07fb64e27d) accidentally removed the memoizing but it should be there.

Reply:
Okay, cool.